client/src/components/previews/OrganizationPreview.js

```diff
@@ -148,9 +148,11 @@ const OrganizationPreview = ({ className, uuid }) => {
             label={Settings.fields.organization.parentOrg}
             value={
               <LinkTo modelType="Organization" model={organization.parentOrg}>
-                {organization.parentOrg.shortName}{" "}
-                {organization.parentOrg.longName}{" "}
-                {organization.parentOrg.identificationCode}
+                {Organization.toIdentificationCodeString(
+                  organization.parentOrg?.shortName,
+                  organization.parentOrg?.longName,
+                  organization.parentOrg?.identificationCode
```

Comment:
Optional chaining (`?.`) is not strictly necessary here, as there's already a guarding condition surrounding this element.